apps/schematic/api/schematic_api/controllers/manifest_validation_controller_impl.py

```diff
@@ -1,56 +1,24 @@
 """Implementation of manifest validation endpoints"""
-import tempfile
-import os
-import io
-import json
+
+# pylint: disable=too-many-locals
+
 from typing import Any
 
-import pandas as pd
 from schematic import CONFIG  # type: ignore
 from schematic.models.metadata import MetadataModel  # type: ignore
+from schematic.utils.schema_utils import DisplayLabelType  # type: ignore
 
 from schematic_api.models.manifest_validation_result import ManifestValidationResult
 from schematic_api.models.basic_error import BasicError
 from schematic_api.controllers.utils import (
     handle_exceptions,
     get_access_token,
     download_schema_file_as_jsonld,
+    save_manifest_json_string_as_csv,
+    save_manifest_csv_string_as_csv,
 )
 
 
-def save_manifest_json_string_as_csv(manifest_json_string: str) -> str:
-    """Takes a manifest json string and converts it to a csv file
-
-    Args:
-        manifest_json_string (str): The manifest in json string form
-
-    Returns:
-        str: The path of the csv file
-    """
-    temp_dir = tempfile.gettempdir()
-    temp_path = os.path.join(temp_dir, "manifest.csv")
-    json_dict = json.loads(manifest_json_string)
-    manifest_df = pd.DataFrame(json_dict)
-    manifest_df.to_csv(temp_path, encoding="utf-8", index=False)
-    return temp_path
-
-
-def save_manifest_csv_string_as_csv(manifest_csv_string: bytes) -> str:
-    """Takes a manifest csv string and converts it to a csv file
-
-    Args:
-        manifest_csv_string (bytes): The manifest in csv string form
-
-    Returns:
-        str: The path of the csv file
-    """
-    temp_dir = tempfile.gettempdir()
-    temp_path = os.path.join(temp_dir, "manifest.csv")
-    manifest_df = pd.read_csv(io.BytesIO(manifest_csv_string), sep=",")
-    manifest_df.to_csv(temp_path, encoding="utf-8", index=False)
-    return temp_path
-
-
 def submit_manifest_with_schematic(  # pylint: disable=too-many-arguments
```

Comment:
I missed this before, but why is submit in the validation controller? Is this because you _can_ validate during submission?

Reply:
Yeah, I grouped them together for that reason. If you think it would make more sense to split them up I'm open to it, but would do it in a different PR.

Reply:
I think as long as its not confusing to users, it probably doesn't matter, I'll leave it up to you.

Reply:
Sounds good to me I'll leave it for now.